src/tools/rbd_mirror/group_replayer/BootstrapRequest.h

```diff
@@ -90,73 +86,28 @@ class BootstrapRequest : public CancelableRequest {
    *
    * <start>
    *    |
-   *    v
-   * GET_REMOTE_GROUP_ID  * * * * * * * * * * *
-   *    |                 (noent)             *
-   *    v                                     v
-   * GET_REMOTE_GROUP_NAME  * * * * * * * * * *
-   *    |                 (noent)             *
-   *    v                                     v
-   * GET_REMOTE_MIRROR_GROUP  * * * * * * * * *
-   *    |           (noent or not primary)    *
-   *    v                                     v
-   * LIST_REMOTE_GROUP_SNAPSHOTS  * * * * * * *
-   *    |                             (noent) *
-   *    v                                     v
-   * LIST_REMOTE_GROUP  * * * * * * * * * * * *
-   *    |  (repeat if neeeded)        (noent) *
-   *    v                                     v
-   * GET_REMOTE_MIRROR_IMAGE  * * * * * * * * *
-   *    |  (repeat for every image)   (noent) *
-   *    |                                     v
-   *    |/< * * * * * * * * * * * * * * * * * *
-   *    v
-   * GET_LOCAL_GROUP_ID * * * * * * * * * * * *
-   *    |               (noent)               *
-   *    v                                     *
-   * GET_LOCAL_GROUP_NAME                     *
+   *    v                           (error)
+   * PREPARE_LOCAL_GROUP  * * * * * * * * * * *
+   *    |                                     *
+   *    v                            (error)  *
+   * PREPARE_REMOTE_GROUP_NAME  * * * * * * * *
+   *    |                                     *
+   *    | (remote dne)                        *
```

Comment:
you mean remote done?

Reply:
DNE== does not exist.

Reply:
Nit: dne is unclear, can you please elaborate it with may be full form, if in case you had to re-spin this PR.

Reply:
As other files use this abbreviation I think it fine to continue with it.